docs/source/techspecs/inputsystem.rst

```diff
@@ -114,8 +114,8 @@ fields:
   and neutral positions.  They are used for analog joystick axes,
   displacement-sensitive pedals, paddle knobs, and other emulated inputs
   with a defined range.
-* **Relative analog fields** have a range with with defined
-  minimum, maximum and starting positions.  On each update, the value
+* **Relative analog fields** have a range with a defined minimum,
+  maximum and starting positions.  On each update, the value
```

Comment:
The “a” is misplaced, it should just be “…with defined minimum, maximum and starting positions.”